pkg/server/handler/v1alpha1/workflowtrigger.go

```diff
@@ -56,8 +56,15 @@ func ListWorkflowTriggers(ctx context.Context, tenant, project, workflow string,
 	}
 
 	items := workflowTriggers.Items
+	var result []v1alpha1.WorkflowTrigger
```

Comment:
I'd prefer to add workflow label, then we can directly filter them by label selector.

Reply:
ok